include/aws/common/atomics_msvc.inl

```diff
@@ -71,6 +88,18 @@ typedef long aws_atomic_impl_int_t;
 typedef long long aws_atomic_impl_int_t;
 #endif
 
+#ifdef _M_ARM64
+#    define RW_BARRIER() __dmb(_ARM64_BARRIER_SY) /* hardare read write barrier */
```

Comment:
This file is included from public headers. These macros need some "namespacing" so they don't collide with anything that might exist in user-code

```suggestion
#    define AWS_RW_BARRIER() __dmb(_ARM64_BARRIER_SY) /* hardware read write barrier */
```

Reply:
debatable: #undef these at the end of the file, so they don't remain part of our public API ???